src/modules/keyboardmanager/common/KeyDelay.cpp

```diff
@@ -0,0 +1,167 @@
+#include "pch.h"
+#include "KeyDelay.h"
+
+#define LONG_PRESS_DELAY_MILLIS 900
+#define ON_HOLD_WAIT_TIMEOUT_MILLIS 50

```

Comment:
We should either keep these as constants in the class rather than using #define, or we can just add them as class members and set them to these values or have it as default values in the constructor. That way we can make this a configurable option. This could be reused if it were shifted to the common library for PowerToys and then have configurable delays would be very useful for the other PowerToys as well.